### ISSUE TITLE
Fix linter issues in example code

### ISSUE THREAD

**Issue by bascrezee** [MeteoSwiss-APN/mch-python-blueprint#38]
While working on #37 it turned out that parts of the cookiecutter code are not flake8 conform. Have to find out why this only popped up now and not before when running the checker without the two config files in the repository.

```
tests/ralmo2ascii/test_cli.py:12:34: A002 argument "exit" is shadowing a python builtin
tests/ralmo2ascii/test_cli.py:22:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:27:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:32:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:36:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:40:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:48:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:52:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:56:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:60:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:70:1: D102 Missing docstring in public method
tests/ralmo2ascii/test_cli.py:74:1: D102 Missing docstring in public method
src/ralmo2ascii/cli.py:12:1: DAR101 Missing parameter(s) in Docstring: - ctx
src/ralmo2ascii/cli.py:12:1: DAR101 Missing parameter(s) in Docstring: - param
src/ralmo2ascii/cli.py:12:1: DAR101 Missing parameter(s) in Docstring: - value
src/ralmo2ascii/cli.py:22:1: DAR101 Missing parameter(s) in Docstring: - **kwargs
src/ralmo2ascii/cli.py:22:1: DAR101 Missing parameter(s) in Docstring: - *args
src/ralmo2ascii/cli.py:22:1: DAR101 Missing parameter(s) in Docstring: - ctx
src/ralmo2ascii/cli.py:59:1: DAR101 Missing parameter(s) in Docstring: - **kwargs
src/ralmo2ascii/cli.py:59:1: DAR101 Missing parameter(s) in Docstring: - ctx
src/ralmo2ascii/cli.py:59:1: DAR101 Missing parameter(s) in Docstring: - number
src/ralmo2ascii/cli.py:61:1: D103 Missing docstring in public function
src/ralmo2ascii/cli.py:70:1: D103 Missing docstring in public function
src/ralmo2ascii/cli.py:78:1: D103 Missing docstring in public function
src/ralmo2ascii/cli.py:86:1: D103 Missing docstring in public function
src/ralmo2ascii/cli.py:94:1: D103 Missing docstring in public function
src/ralmo2ascii/mutable_number.py:10:1: DAR101 Missing parameter(s) in Docstring: - number
src/ralmo2ascii/mutable_number.py:19:1: DAR101 Missing parameter(s) in Docstring: - idx
src/ralmo2ascii/mutable_number.py:19:1: DAR201 Missing "Returns" in Docstring: - return
src/ralmo2ascii/mutable_number.py:29:1: DAR101 Missing parameter(s) in Docstring: - addend
src/ralmo2ascii/mutable_number.py:29:1: DAR201 Missing "Returns" in Docstring: - return
src/ralmo2ascii/mutable_number.py:35:1: DAR201 Missing "Returns" in Docstring: - return
src/ralmo2ascii/mutable_number.py:35:1: DAR101 Missing parameter(s) in Docstring: - subtrahend
src/ralmo2ascii/mutable_number.py:41:1: DAR101 Missing parameter(s) in Docstring: - factor
src/ralmo2ascii/mutable_number.py:41:1: DAR201 Missing "Returns" in Docstring: - return
src/ralmo2ascii/mutable_number.py:47:1: DAR101 Missing parameter(s) in Docstring: - divisor
src/ralmo2ascii/mutable_number.py:47:1: DAR201 Missing "Returns" in Docstring: - return
tests/ralmo2ascii/test_mutable_number.py:6:1: D103 Missing docstring in public function
tests/ralmo2ascii/test_mutable_number.py:11:1: D103 Missing docstring in public function
tests/ralmo2ascii/test_mutable_number.py:17:1: D103 Missing docstring in public function
tests/ralmo2ascii/test_mutable_number.py:23:1: D103 Missing docstring in public function
tests/ralmo2ascii/test_mutable_number.py:29:1: D103 Missing docstring in public function
tests/ralmo2ascii/test_mutable_number.py:35:1: D103 Missing docstring in public function
tests/ralmo2ascii/test_utils.py:9:1: D103 Missing docstring in public function
```

I think these files should be excluded from the checker.

**Comment by regDaniel** [comment]
Add linting to either Jenkins or GH actions.

**Comment by ruestefa** [comment]
Turns out the main problem was not the code as such, but that `pylint` doesn't pick up the config in `tox.ini` unless asked to do so with `--rcfile=tox.ini`. (`flake8` doesn't have any complaints in the first place.) Passing the flag (or moving the config to `pyproject.toml`) removes almost all errors.

**Comment by ruestefa** [comment]
Fixed in #48.
